trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/index.ts

```diff
@@ -32,6 +32,7 @@ export const getCreateRecordCorrectionRequest = async (req: GetCreateRecordCorre
     const { reportId, feeRecordId } = req.params;
 
     const feeRecord = await api.getFeeRecord(reportId, feeRecordId, userToken);
+    const formValues = await api.getFeeRecordCorrectionTransientFormData(reportId, feeRecordId, user, userToken);
```

Comment:
do we need to check that anything has been returned before rendering?

Reply:
I don't believe we're doing this handling for other controllers from a quick search. The API returns either an empty object or an object with form values. If it returns something like `null` or `undefined` (for some reason) the page should handle these still and present an empty form.

I feel conflicted about adding a `!formValues` check in given we're also controlling the API it's calling. Further, we wouldn't be able to add any tests for such a check (due to the typing on the api function docs), it'd just be tested through the other functions (as not throwing).